templates/Dockerfile.erb

```diff
@@ -18,12 +18,14 @@ ENV FLUENTD_DISABLE_BUNDLER_INJECTION 1
 
 COPY Gemfile* /fluentd/
   RUN buildDeps="sudo make gcc g++ libc-dev libffi-dev<% if target == "graylog" %> build-essential patch zlib1g-dev liblzma-dev<% elsif target == "kafka" %> build-essential autoconf automake libtool pkg-config<% end %><% if requires_git %> git<% end %>" \
+  runtimeDeps="<% if target == "kafka" %>krb5-kdc libsasl2-modules-gssapi-mit libsasl2-dev <% end %>" \
+     && export DEBIAN_FRONTEND=noninteractive \
```

Comment:
Why add this envvar?
Executing `ldd $(gem contents rdkafka | grep librdkafka.so) | grep libsasl2.so.2` will be blocked?

Reply:
No, it's krb5 which is blocking.  I had that discussion offline.

Reply:
The commit message should be explainjng the problem.

Reply:
Ah, I missed the commmit message. If this is only for kafka image, it should be in `<% if target == "kafka" %>`.

Reply:
Done. Note that I have added it into a new if block as I want a complete separation from the runtimeDeps section